spec/client_spec.js

```diff
@@ -173,6 +231,33 @@ describe('Client', function() {
 
     });
 
+    describe('#get', function() {
+      beforeEach(function() {
+        sandbox.spy(window, 'setTimeout');
+      });
+
+      it('returns a promise', function() {
+        expect(subject.get('ticket.subject')).to.be.a.promise;
```

Comment:
It's a little hacky, but the way we've been triggering the message event is by [stubbing `window.addEventListener`](https://github.com/zendesk/zendesk_app_framework_sdk/blob/adammw/sdk-v2/spec/client_spec.js#L12) and then [calling the argument](https://github.com/zendesk/zendesk_app_framework_sdk/blob/adammw/sdk-v2/spec/client_spec.js#L68) it receives, which is the [`messageHandler`](messageHandler). This way you can pass an event in the format the sdk expects and test that it resolves the promise.